rpc-server/src/config.rs

```diff
@@ -1,12 +1,43 @@
 use crate::modules::blocks::{CacheBlock, FinalBlockInfo};
 use futures::executor::block_on;
 
+#[derive(Debug, Clone)]
+pub struct GenesisInfo {
```

Comment:
Would you be so kind as to add for the future us the doc-string explaining what this struct does? Thank you!